assembly/assembly-main/src/assembly/bin/che-install-plugin.sh

```diff
@@ -194,7 +194,7 @@ function set_environment_variables {
   # The machine web app project that will be built with the extension
   PLUGIN_WSAGENT_WAR_DIR="${SDK_DIR}/assembly-wsagent-war"
 
-  # Creates the ws-agent.zip artifact for a Che assembly, which packages Tomcat + machine war into single package
```

Comment:
What about `che-install-plugin.bat` will it work on windows after changes in this PR?